website/blog/2024-02-28-maci-v1.2.0.md

```diff
@@ -0,0 +1,140 @@
+---
+slug: maci-v1-2-0-release
+title: MACI v1.2.0 Release
+authors:
+  name: ctrlc03
+  title: MACI team
+  url: https://pse.dev
+  image_url: /img/pse-logo-round.png
+tags: [release, audit, security, documentation]
+---
+
+# MACI v1.2 Release
+
+We are pleased to announce the release of MACI [v1.2.0](https://github.com/privacy-scaling-explorations/maci/releases/tag/v1.2.0)!
+
+This is our first release since [MACI v1.1.1](/blog/maci-v1-1-1-release) just over one year ago. This new release focuses on improved developer experience, security, performance and clearer documentation for users looking to learn and use MACI.
+
+## Background
```

Comment:
I know this is a draft PR, but thinking about this part is actually same as the one in v1.1.1, should we just remove it? I think the most important part in this v1.2.0 report is 1) which parts we refactored? 2) why do the refactors and 3) how they affect usage?

Reply:
Fair callout, I do think worth asking if this should be included in every release post or not. Maybe instead we should just reference the Introduction page for this type of background info?

Reply:
I'd say for now my (loosely held) opinion is to keep it in, given that we call out
- the quadratic & non-quadratic nature of it
- clr.fund is in the process of upgrading to v1.2

But I could be easily convinced to update both release posts to just reference this info on the Introduction page